common/djangoapps/student/views.py

```diff
@@ -1855,6 +1857,16 @@ def _enroll_user_in_pending_courses(student):
                 )
 
 
+def _record_registration_attribution(request, user):
+    """
+    Attribute this user's registration to the referring affiliate, if
+    applicable.
+    """
+    affiliate_id = request.COOKIES.get(settings.AFFILIATE_COOKIE_NAME)
+    if user is not None and affiliate_id is not None:
+        UserAttribute.set_user_attribute(user, settings.AFFILIATE_COOKIE_NAME, affiliate_id)
```

Comment:
I just noticed that the attribute name is dependent on the cookie name. The attribute name should be a constant value (e.g. `'registration_affiliate_id'`). @peter-fogg can you patch the RC, please?